dts/riscv/ite/it8xxx2.dtsi

```diff
@@ -819,6 +819,48 @@
 			#gpio-cells = <2>;
 		};
 
+		gpioksi: gpiokscan@f01d07 {
+			compatible = "ite,it8xxx2-gpiokscan";
+			reg = <0x00f01d07 1
+			       0x00f01d06 1
+			       0x00f01d08 1
+			       0x00f01d09 1
+			       0x00f01d26 1>;
+			reg-names = "KSIGOEN", "KSIGCTRL", "KSIGDAT", "KSIGDMRR",
+				    "KSIGPODR";
```

Comment:
Use lower case for the reg-names assignments.  Also use the common suffix `goen`, `gctrl`, instead of the full register name.

Reply:
Done